packages/functional-tests/src/tests/animation-dynamic-test.ts

```diff
@@ -0,0 +1,93 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as MRE from '@microsoft/mixed-reality-extension-sdk';
+
+import { Test } from '../test';
+
+const RealtimeAnimData: MRE.AnimationDataLike = {
+	tracks: [{
+		target: MRE.ActorPath("target").transform.app.position,
+		easing: MRE.AnimationEaseCurves.EaseOutQuadratic,
+		keyframes: [{
+			time: 0,
+			value: MRE.ActorPath("anchor1").transform.app.position
+		}, {
+			time: 1,
+			value: MRE.ActorPath("anchor2").transform.app.position
+		}, {
+			time: 2,
+			value: MRE.ActorPath("anchor3").transform.app.position
+		}, {
+			time: 3,
+			value: MRE.ActorPath("anchor1").transform.app.position
+		}]
+	}]
+};
+
+export default class AnimationDynamicTest extends Test {
```

Comment:
The boxes do not move for me:
![image](https://user-images.githubusercontent.com/12176807/79381914-89bc0f00-7f17-11ea-8cf5-0fe3f4e378a1.png)

Reply:
The boxes are grabbable, they don't move on their own.

Reply:
I thought I tried that... will try again.